pkg/distributor/distributor.go

```diff
@@ -251,6 +258,44 @@ const (
 	limitLabel               = "limit"
 )
 
+type PushMetrics struct {
+	OTLPRequestCounter   *prometheus.CounterVec
+	UncompressedBodySize *prometheus.HistogramVec
```

Comment:
These don't need to be exported anymore, right?

Reply:
_changed @ 7b250900dc798b6b5ea543affe5c8a5dc8c0daf1_